plugins/scenario/hu.bme.mit.gamma.scenario.statechart.generator/src/hu/bme/mit/gamma/scenario/statechart/generator/StatechartGenerator.xtend

```diff
@@ -702,12 +737,25 @@ class StatechartGenerator extends ScenarioModelSwitch<EObject> {
 		assign.lhs = refe
 		return assign
 	}
+	
+	def VariableDeclaration getOrCreate(HashMap<String, VariableDeclaration> map, String string) {
```

Comment:
Always use interfaces when declaring collection types, not classes, e.g., `Map<String...`